bitnami/jupyterhub/templates/_helpers.tpl

```diff
@@ -71,6 +71,26 @@ Return the CryptKeeper value
     {{- end }}
 {{- end }}
 
+{{/*
+Return a service API token
+*/}}
+{{- define "jupyterhub.hub.services.get_api_token" -}}
+    {{- $_ := index . 0 }}
+    {{- $service_key := index . 1 }}
```

Comment:
If you use a dictionary instead of a list, it would be a little bit clearer and easier to understand.